tool-plugins/intellij/src/test/java/io/ballerina/formatting/BallerinaFormattingTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.ballerina.formatting;
+
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.psi.codeStyle.CodeStyleManager;
+import io.ballerina.BallerinaCodeInsightFixtureTestCase;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.netty.util.internal.StringUtil.EMPTY_STRING;
+
+/**
+ * Formatting tests.
+ */
+public class BallerinaFormattingTest extends BallerinaCodeInsightFixtureTestCase {
+
+    @Override
+    protected String getTestDataPath() {
+        return "src/test/resources/testData/formatting/BBE";
+    }
+
+    //this test validates the formatting for the ballerina-by-examples
```

Comment:
Capitalize the first letter and add a space after that.

Reply:
Done

Reply:
Where exactly you have done this?